raspberry_pi/app/camera/interpreter_factory.py

```diff
@@ -64,26 +64,15 @@ def get(
                     "tag" + str(camera_num),
                 )
                 return TagDetector(identity, cam, camera_num, display, network)
-            case (Identity.DEV):
-                display = RealDisplay(
-                    int(scale * size.width),
-                    int(scale * size.height),
-                    "tag" + str(camera_num),
-                )
-                # return NullDetector(cam, display)
-                return TagDetector(identity, cam, camera_num, display, network, False)
-            case (
-                # Identity.DEV
-                Identity.CORAL_RIGHT
-                  | Identity.CORAL_LEFT):
+            case (Identity.DEV|Identity.DEV2| Identity.CORAL_RIGHT| Identity.CORAL_LEFT):
                 display = RealDisplay(
                     int(scale * size.width),
                     int(scale * size.height),
                     "combined" + str(camera_num),
                 )
                 # GREEN TARGET VALUES
-                object_lower = np.array((40, 50, 100))
-                object_higher = np.array((70, 255, 255))
+                object_lower = np.array((0, 0, 150))
```

Comment:
notice the comment here that says "GREEN TARGET" ... i'm pretty sure that you were not using a green target to get these numbers.  you should change the comment to say what you *were* doing.  or maybe comment out the green values (since they might be useful someday) and then add the white values, with a comment that says how you obtained these magic numbers.